styles.css

```diff
@@ -0,0 +1,81 @@
+@import url('https://fonts.googleapis.com/css2?family=Playfair+Display:wght@400;700&family=Poppins&display=swap');
+
+* {
+  box-sizing: border-box;
+  max-width: 100%;
+  margin: 0;
+  padding: 0;
+}
+
+.nav-bar {
+  display: flex;
+  justify-content: space-around;
+  padding: 24px;
+  margin-top: 58px;
+}
```

Comment:
- [ ] Currently, your nav-bar seems to have different paddings as compared to the Figma template. I've marked the areas with excess padding with red lines. The values for margins and padding in the Figma template are not for browsers, so please try to go by intuition and adjust the padding accordingly.
> Your Design
![image](https://user-images.githubusercontent.com/90556221/170391570-b925a2d1-93a3-4cf2-9981-2d9537ddc376.png)
> Figma Template
![image](https://user-images.githubusercontent.com/90556221/170391636-074f7b1a-ae37-4734-83a4-0c420e11dec1.png)